lib/domain/db/models/timeline_pin_type.dart

```diff
@@ -1,13 +1,13 @@
-enum TimelinePinType { NOTIFICATION, PIN, REMINDER }
+enum TimelinePinType { notification, pin, reminder }
 
 extension ProtocolExtension on TimelinePinType {
   int toProtocolNumber() {
     switch (this) {
```

Comment:
Same comment as for TimelinePinLayout

Reply:
These numbers must be constant since they correspond to integers that get sent to the watch. I don't think generating them is a good idea since generator can change values.